src/lib/configwidgetslib/erroroverlay.ui

```diff
@@ -211,6 +211,33 @@
      </property>
     </spacer>
    </item>
+   <item>
+    <widget class="QLabel" name="runfcitxLabel">
+     <property name="text">
+      <string/>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="runfcitxButton">
```

Comment:
runFcitxButton

Reply:
Also can you add an icon to the button? You may use system-run as icon name.

Reply:
ok